openlibrary/templates/type/edition/view.html

```diff
@@ -424,7 +424,7 @@ <h3 class="list-header collapse">$_("The Physical Object")</h3>
 
 
           <div class="section">
-            <h3 class="list-header collapse">$_("ID Numbers")</h3>
+            <h3 class="edition-header">$_("ID Numbers")</h3>
```

Comment:
These should still be list-header.

The only issue is with the generic h3's that don't match list-header.

Reply:
list-header or previous list-header collapse?

Reply:
The html for these is fine so retain as is.